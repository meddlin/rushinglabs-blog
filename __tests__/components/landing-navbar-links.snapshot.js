import { render } from '@testing-library/react';
import LandingNavbarLinks from '../../components/landing/landing-navbar-links';

it('renders homepage unchanged', () => {
    const { container } = render(<LandingNavbarLinks />);

  expect(container).toMatchSnapshot()
})