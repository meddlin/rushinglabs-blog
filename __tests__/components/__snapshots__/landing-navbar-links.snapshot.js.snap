// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="flex flex-row navbarLinks"
  >
    <h2>
      <a
        class="text-xl font-bold no-underline text-inherit"
        href="/blog"
      >
        Blog
      </a>
    </h2>
    <h2>
      <a
        class="text-xl font-bold no-underline text-inherit"
        href="/projects"
      >
        Projects
      </a>
    </h2>
    <h2>
      <a
        class="text-xl font-bold no-underline text-inherit"
        href="https://www.youtube.com/@rushinglabs"
      >
        YouTube
      </a>
    </h2>
    <h2>
      <a
        class="text-xl font-bold no-underline text-inherit"
        href="/about"
      >
        About
      </a>
    </h2>
  </div>
</div>
`;
